syntax = "proto3";

package loan;

message MsgLog {
  Msg_Type msg_type = 1;
  int32 msg_cmd = 2;
  string service_name = 3;
  string LogContents = 4;

  enum Msg_Type{
    SEND_LOG = 0;
    SEND_COLLECT = 1;
  }
}
